CHANGELOG.rst

```diff
@@ -0,0 +1,43 @@
+=========
+Changelog
+=========
+
+######
+Future
+######
```

Comment:
I think this version should be 1.0 :muscle:

Reply:
I will update this when I bump the version to 1.0.0 in #16 .